build_osx.md

```diff
@@ -27,7 +27,7 @@ can now be built.
 
 ```
 $ cd go/src/github.com/containers/podman
-$ make podman-remote-darwin
+$ make podman-remote
 $ mv bin/darwin/podman bin/podman
```

Comment:
Does the executable still end up in bin/darwin/podman?

Reply:
yes, it does:

```sh
% tree bin  
bin  [error opening dir]

0 directories, 0 files
```

```sh
$ make podman-remote 
Makefile:138: invalid `override' directive
mkdir -p bin/darwin
CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build \
                 \
                -ldflags '-X github.com/containers/podman/v4/libpod/define.gitCommit=cfa651f8019a665d5baa3dd738b1d76f00d9a861 -X github.com/containers/podman/v4/libpod/define.buildInfo=1668021959 -X github.com/containers/podman/v4/libpod/config._installPrefix=/usr/local -X github.com/containers/podman/v4/libpod/config._etcDir=/usr/local/etc -X github.com/containers/common/pkg/config.additionalHelperBinariesDir= -X github.com/containers/podman/v4/pkg/quadlet.QuadletUserName=quadlet ' \
                -tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp" \
                -o bin/darwin/podman ./cmd/podman
```


```sh
$ tree bin          
bin
`-- darwin
    `-- podman

1 directory, 1 file
```

Reply:
Awesome.